portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,7 +27,6 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> quotes;
   Gson gson = new Gson();
   private final static ImmutableList<String> QUOTES = ImmutableList.of
```

Comment:
nit: move ImmutableList.of to next line and break on the text.

ImmutableList.of(
  "A ship in port....